src/ngraph/runtime/cpu/pass/cpu_layout.cpp

```diff
@@ -377,6 +377,25 @@ namespace ngraph
                     }
                 }
 
+                template <>
+                void CPULayout::LAYOUT_DECL(ngraph::op::ConvolutionBiasRelu)
+                {
```

Comment:
I wonder if this logic is similar to `ConvolutionRelu` and could be refactored into a helper?